### PR TITLE
[Laravel] Add Caching documentation

### PR DESCRIPTION
**Pull request by toitzi**
Relates to this PR: https://github.com/api-platform/core/pull/6751

### CODE REVIEW

File: core/upgrade-guide.md [hunk 1]
```diff
@@ -93,7 +93,7 @@ For new users we recommend to use
 event_listeners_backward_compatibility_layer: false
 ```
 
-This allows API Platform to not use http kernel event listeners. It also allows you to force options like `read: true` or `validate: true`. This simplifies use cases like [validating a delete operation](/docs/v3.2/guides/delete-operation-with-validation/)
+This allows API Platform to not use http kernel event listeners. It also allows you to force options like `read: true` or `validate: true`. This simplifies use cases like [validating a delete operation](https://api-platform.com/docs/v3.2/guides/delete-operation-with-validation/)
```
Comment (soyuka) [inline core/upgrade-guide.md:96]:
```suggestion
This allows API Platform to not use http kernel event listeners. It also allows you to force options like `read: true` or `validate: true`. This simplifies use cases like [validating a delete operation](/docs/v3.2/guides/delete-operation-with-validation/)
```

**Comment by soyuka** [comment]
oh my comment is probably wrong, I'd `git reset --hard upstream/4.0` then cherry-pick [036d616](https://github.com/api-platform/docs/pull/2048/commits/036d61680b101ec21e70d6104a5611bb6c0515a6)

**Comment by toitzi** [comment]
yeah one moment...

**Comment by toitzi** [comment]
@soyuka done, should be fine now

**Comment by soyuka** [comment]
thanks!

**Pull request merged**